inbm-lib/tests/unit/inbm_lib/test_version.py

```diff
@@ -1,6 +1,6 @@
 from unittest import TestCase
-from inbm_lib.version import get_inbm_version, get_inbm_commit, get_inbm_vision_host_version, get_inbm_vision_host_commit, get_inbm_vision_node_version, get_inbm_vision_node_commit
-from inbm_lib.version import get_friendly_inbm_version_commit, get_friendly_inbm_vision_version_commit
+from inbm_lib.version import get_inbm_version, get_inbm_commit
+from inbm_lib.version import get_friendly_inbm_version_commit
```

Comment:
**<h2>$\color{#FFA500}Improvements: Code Refactoring$</h2>**
**Suggestion:** Refactor the functions to use regular expressions or other parsing methods that are more robust against variations in the version text format.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
# Suggested refactoring in inbm_lib.version module
import re

VERSION_PATTERN = re.compile(r"Version:\s*(\S+)")
COMMIT_PATTERN = re.compile(r"Commit:\s*(\S+)")

def get_inbm_version() -> Optional[str]:
    contents = _read_inbm_version_file()
    if contents is None:
        return None
    match = VERSION_PATTERN.search(contents)
    return match.group(1) if match else None

def get_inbm_commit() -> Optional[str]:
    contents = _read_inbm_version_file()
    if contents is None:
        return None
    match = COMMIT_PATTERN.search(contents)
    return match.group(1) if match else None
```